addons/website_slides_survey/models/survey_survey.py

```diff
@@ -1,11 +1,42 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import models, fields, api
 
 
 class Survey(models.Model):
     _inherit = 'survey.survey'
+    slide_ids = fields.One2many('slide.slide', 'survey_id', 'Certifications')
+    slide_channel_ids = fields.One2many('slide.channel', compute='_compute_slide_channel_ids', groups='website_slides.group_website_slides_officer')
+    slide_channel_count = fields.Integer("Linked Courses", compute='_compute_slide_channel_ids', groups='website_slides.group_website_slides_officer')
+
+    @api.depends('slide_ids.channel_id')
+    def _compute_slide_channel_ids(self):
+        for survey in self:
+            if survey.slide_ids:
+                survey.slide_channel_ids = survey.slide_ids.mapped('channel_id')
+            else:
+                survey.slide_channel_ids = self.env['slide.channel']
+            survey.slide_channel_count = len(survey.slide_channel_ids)
+
+    # ---------------------------------------------------------
+    # Actions
+    # ---------------------------------------------------------
+
+    def action_survey_view_slide_channels(self):
+        action = self.env.ref('website_slides.slide_channel_action_overview').read()[0]
+        action.update({'display_name': "Courses"})
```

Comment:
The translations are handled automatically when using strings in :
- a field attribute like `string` or `help`
- a XML node
- ...

But when using plain strings in the python/JS code, you have to handle to translation manually using the underscore function.

So here it will be `action['display_name'] = _("Courses")`

(You also have to import _ of course `from odoo import _`)

Reply:
Understood, fixed it.